index.js

```diff
@@ -1,3 +1,27 @@
+const { EmailClient } = require('./entities/EmailClient');
+// const { SlackCLient } = require('./entities/SlackClient');
+const { getAlertInfo, getTestId } = require('./lib/db/db_query');
+require('dotenv').config();
+
 exports.handler = async (event) => {
-  console.log('EVENT OBJ: ', event);
+  const { id: testId } = await getTestId(event.title);
```

Comment:
What do you think about removing this query and just including a join to the tests table in `getAlertInfo`? You'd then just pass the `event.title` and use that for the WHERE condition

Reply:
you have great SQL instincts!

```
async function getAlertInfo(testTitle) {
  const query = `
    SELECT t.id AS test_id, a.type, a.destination
    FROM tests as t
    INNER JOIN tests_alerts as ta
        on ta.test_id = t.id
    INNER JOIN alerts as a
        on a.id = ta.alerts_id
    WHERE t.name = $1
  `;
  const result = await dbQuery(query, testTitle);
  return result.rows;
}

```